index.bs

```diff
@@ -1099,18 +1099,25 @@ optional |expires|,
 |sameSite|,
 run the following steps:
 
+1. If |name|, |value|, |domain|, or |path| contain U+003B (`;`), any [=C0 control=] character, or U+007F, then return failure.
 1. If |name|'s [=string/length=] is 0 and |value| contains U+003D (`=`), then return failure.
 1. If |name|'s [=string/length=] is 0 and |value|'s [=string/length=] is 0, then return failure.
+1. Determine whether |name| and |value| together exceed cookie size limits using these steps:
+    1. [=UTF-8 encode=] |name| and then compute its [=byte sequence=] [=byte sequence/length=].
+    1. [=UTF-8 encode=] |value| and then compute its [=byte sequence=] [=byte sequence/length=].
+    1. If the sum of the two computed lengths is greater than 4096 [=byte|bytes=], then return failure.
```

Comment:
To avoid these magic numbers (4096 here, 1024 below), would it make sense to define "constants" for them, referencing the RFC?

Reply:
Thanks for the thorough feedback! 💯

For this one, are you thinking I should do something like:
```
1.  Let |namePlusValueMaxSize| be 4096 [=byte|bytes=] (as specified in [[RFC6265bis#section-5.5|RFC6265bis]]).
...
1.  If the sum of the two computed lengths is greater than |namePlusValueMaxSize| [=byte|bytes=], then return failure.
```

or is there a more elegant way to achieve this?

Reply:
For the constant, I'd do a <dfn> earlier in the spec... in the cookie-concept section which acts as a bridge to the RFC, maybe something like:

`The <dfn for=cookie>maximum name plus size</dfn> for a cookie is 4096, per Cookies: HTTP State blah blah blah`

and then in the algorithm:

`If the [=byte sequence=] [=byte sequence/length=] of |encodedName| plus the [=byte sequence=] [=byte sequence/length=] of encodedValue is greater than the [=cookie/maximum name plus size=], then return failure.`